test/integration/mentor_configure_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class MentorConfigureTest < ActionDispatch::IntegrationTest
+  test "mentors configuring can see what tracks they already use" do
+    user = create :user
+    user.confirm
+
+    track_one = create :track
+    track_two = create :track
+    user.mentored_tracks = [track_one]
+
+    sign_in! user
```

Comment:
As an FYI, you can do `sign_in!` and then use `@current_user`, rather than having to set up and confirm the user yourself.